bin/kuzzle-createFirstAdmin.js

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env node
```

Comment:
Many indentation problems in this file. Maybe asking jshint to scan the `bin/` directory would help detect this kind of formating problem.